src/GraphQL/GraphQLExtensions.cs

```diff
@@ -237,7 +239,7 @@ public static IEnumerable<string> IsValidLiteralValue(this IGraphType type, IVal
                     errors.AddRange(result.Select(err => $"In field \"{field.Name}\": {err}"));
                 }
 
-                return errors;
+                return errors.ToArray();
```

Comment:
In theory, this adds allocations, but in practice there should still be less of them due to the removal of `ToList` from other places 😕